daemons/fenced/fenced_remote.c

```diff
@@ -526,11 +538,12 @@ remote_op_done(remote_fencing_op_t * op, xmlNode * data, int rc, int dup)
         level = LOG_NOTICE;
     }
 
-    do_crm_log(level, "Operation '%s'%s%s on %s for %s@%s.%.8s: %s",
+    do_crm_log(level, "Operation '%s'%s%s on %s for %s@%s.%.8s: %s%s",
                op->action, (op->target? " targeting " : ""),
                (op->target? op->target : ""),
                (op->delegate? op->delegate : "<no-one>"),
-               op->client_name, op->originator, op->id, pcmk_strerror(rc));
+               op->client_name, op->originator, op->id, pcmk_strerror(rc),
+               (op_merged? "(merged)" : ""));
```

Comment:
In each case, a space before (merged) would help separate it. In this case, I think it might be a good idea to put (merged) before the ':' so users don't think it's part of the error message.

Reply:
Hi Ken,

Thanks for your comment.

Okay!
In your opinion, it would be the following log, is it correct?

```
Dec 17 09:29:23 rh77hv-02 pacemaker-fenced[30658]: notice: Operation 'reboot' targeting rh77hv-01 on rh77hv-02 for pacemaker-controld.30662@rh77hv-02.750c4fa9(merged): OK
```

I will wait for some more opinions from others.

Best Regards,
Hideo Yamauchi.

Reply:
Hi Hideo,

Yes, that is what I was thinking, though I'd put a space before it.

Reply:
Hi Ken,

Okay!

It is hard to see, but I put a space before "(merged)".
```
Operation 'reboot' targeting rh77hv-01 on rh77hv-02 for pacemaker-controld.30662@rh77hv-02.750c4fa9 (merged): OK
```

Many thanks,
Hideo Yamauchi.

Reply:
Hi Hideo, that looks good

Reply:
Hi Ken, Okay!